packages/babel-preset-env/src/index.ts

```diff
@@ -126,53 +126,47 @@ export const getModulesPluginNames = ({
   shouldTransformESM,
   shouldTransformDynamicImport,
   shouldTransformExportNamespaceFrom,
-  shouldParseTopLevelAwait,
 }: {
   modules: ModuleOption;
   transformations: ModuleTransformationsType;
   shouldTransformESM: boolean;
   shouldTransformDynamicImport: boolean;
   shouldTransformExportNamespaceFrom: boolean;
-  shouldParseTopLevelAwait: boolean;
```

Comment:
TLA is now enabled by default in our parser, so we can unconditionally add the syntax plugin for older parser versions.